app/templates/signup.html

```diff
@@ -27,7 +34,14 @@ <h1>Sign Up & Register</h1>
             {% for error in form.password.errors %}
             <span style="color: red">[{{ error }}]</span>
```

Comment:
nit: avoid inline styles. move all styling to CSS files

Reply:
I see i believe this was added by Don who referenced the flask ppt made by Portier, I think it should be okay but we can remove it